java/client/build.gradle

```diff
@@ -180,7 +181,7 @@ publishing {
         mavenJava(MavenPublication) {
             from components.java
             groupId = 'software.amazon.glide'
-            artifactId = 'glide'
+            artifactId = 'glide-for-redis'
             version = "255.255.255"
```

Comment:
Shall we change the version here as well?

Reply:
version is sed by the workflow

Reply:
added a comment to explain better